MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+## Overview
+
+This document contains a list of maintainers in this repo.
+See [GOVERNANCE.md](GOVERNANCE.md) that explains the function of this file.
+
+## Current Maintainers
+
+| Maintainer          | GitHub ID                                       | Affiliation |
+| ------------------- | ----------------------------------------------- | ----------- |
+| Madelyn Olson       | [madolson](https://github.com/madolson)         | Amazon      |
+| Viktor Zuiderkwast  | [zuiderkwast](https://github.com/zuiderkwast)   | Ericsson    |
+| Zhu Binbin          | [enjoy-binbin](https://github.com/enjoy-binbin) | Tencent     |
+| Wen Hui             | [hwware](https://github.com/hwware)             | Huawei      |
+| Zhao Zhao           | [soloestoy](https://github.com/soloestoy)       | Alibaba     |
+| Ping Xie            | [pingxie](https://github.com/pingxie)           | Google      |
```

Comment:
Can we list the maintainer sorted by Maintainer first character， OR GitHub ID, OR Affiliation? Now it looks like a little bit messy.

Reply:
Will do by github ID, which I will say is the most natural to lexicographically order by.